assets/less/hel/commentlist.less

```diff
@@ -0,0 +1,129 @@
+.sortable-comment-list {
+  &__count {
+    text-align: center;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    padding-top: 30px;
+    border-top: 1px solid @border-color;
+  }
+
+  &__loader {
+    text-align: center;
+  }
+
+  .plugin-frame {
+      width: 100%;
+      height: 450px;
+      border: 0;
+  }
+}
+
+.commentlist {
+  .hearing-comment {
+      margin-top: 30px;
+      border-top: 1px solid #eaeaea;
+      padding-top: 30px;
+
+      &-body {
+          padding-top: 15px;
+          font-size: 1.4rem;
+      }
+      &-votes {
+          float: right;
+          width: 20%;
+          text-align: right;
+      }
+      &-publisher {
+          width: 80%;
+          float: left;
+          font-weight: bold;
+      }
+      &-user {
+          margin-right: 10px;
+          @media (max-width: @screen-xs-max) {
+              display: block;
+          }
+          &-registered {
+              color: #999;
+              :not(.fa)  {
+                  font-size: 1.4rem;
+              }
+          }
+      }
+      &-date {
+          color: #999;
+          font-size: 1.4rem;
+          @media (max-width: @screen-xs-max) {
+              display: block;
+          }
+          @media (min-width: @screen-sm-min) {
+              &:before {
+                  content: "\2022";
+                  margin-right: 10px;
+                  color: #aaa;
+              }
+          }
+      }
+      &-vote-link {
+          padding: 0;
+          border: 0;
+          background: transparent;
+          color: @link-color;
+          letter-spacing: .03em;
+          &:hover, &:focus {
+              background: transparent;
+              color: @brand-primary;
+          }
+          .fa {
+              margin-right: 2px;
+              font-size: 1.4rem;
+          }
+      }
+  }
+
+  .comment-form {
+      background-color: @hel-yellow;
+      padding: 1em;
+
+      &__file {
+          margin: 10px 0;
+          padding-top: 10px;
+      }
+
+      &__image-too-big {
+          color: red !important;
+          padding-bottom: 10px;
+      }
+
+      &__select-button {
+          padding: 10px 0;
+      }
+
+      &__selected-images {
+          padding-top: 10px;
+      }
+  }
+
+  .comment-conditions {
+      padding-top: 15px;
+      font-size: 1.4rem;
+  }
+
+  .section-title {
```

Comment:
For some reason, this doesn't seem to be working. At least the cursor is wrong and the # comments doesn't float right.